client/components/data/domain-management/index.jsx

```diff
@@ -65,7 +65,7 @@ var DomainManagementData = React.createClass( {
 	render: function() {
 		return (
 			<StoreConnection
-				component={ this.props.component }
+				component={ this.props.component.default || this.props.component }
```

Comment:
In order to not break the list tests I export both the raw domain list and the connected list. This component assumed that the module only had one export.